sys/crypto/modes/ccm.c

```diff
@@ -35,7 +35,7 @@ static inline int min(int a, int b)
 }
 
 int ccm_compute_cbc_mac(cipher_t* cipher, uint8_t iv[16],
```

Comment:
The `iv[16]` here can also be `const`. Do you mind piggy backing that or do you think that it is unnecessary optimization here?

Reply:
I only put the minimal ones that were required to make my tests pass, I did not even tried to understand :p
But sure, I will add it.

Reply:
I didn't really check if it makes sense here, I just noticed that it wasn't modified by the function :)

Reply:
Just to be sure, can it be an issue, in general case, for the called function that `iv` and `mac` can be the same pointer when `iv` is said const ?

Here as it is an internal function, it could even be replaced by only one "output" pointer that should contain the "iv", but that is another topic.